elasticsearch/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.elastic.co/elasticsearch/elasticsearch-oss:7.7.1
+FROM docker.elastic.co/elasticsearch/elasticsearch-oss:7.8.0
```

Comment:
We probably want to move this to a separate PR though, right?

Reply:
yep, I think so.